test/test_fingerprint_api.py

```diff
@@ -113,6 +122,34 @@ def test_get_visits_correct_data(self):
         self.api.get_visits(mock_file1)
         self.api.get_visits(mock_file2)
 
+    def test_get_visits_error_403(self):
+        """Test checks correct code run result in case of 403 error"""
+        mock_pool = MockPoolManager(self)
+        self.api.api_client.rest_client.pool_manager = mock_pool
+        mock_file = 'visits_403_error.json'
+        mock_pool.expect_request('GET', TestFingerprintApi.get_get_visits_method_path(visitor_id=mock_file),
+                                 fields=[self.integration_info], headers=self.request_headers,
+                                 preload_content=True, timeout=None, status=403)
+        with self.assertRaises(KnownApiException) as context:
+            self.api.get_visits(mock_file)
+        self.assertEqual(context.exception.status, 403)
+        structured_error = context.exception.structured_error
+        self.assertIsInstance(context.exception.structured_error, ErrorVisits403)
+
+    def test_get_visits_error_429(self):
+        """Test checks correct code run result in case of 429 error"""
+        mock_pool = MockPoolManager(self)
+        self.api.api_client.rest_client.pool_manager = mock_pool
+        mock_file = 'visits_too_many_requests_error.json'
+        mock_pool.expect_request('GET', TestFingerprintApi.get_get_visits_method_path(visitor_id=mock_file),
+                                 fields=[self.integration_info], headers=self.request_headers,
+                                 preload_content=True, timeout=None, status=429)
+        with self.assertRaises(KnownApiException) as context:
+            self.api.get_visits(mock_file)
+        self.assertEqual(context.exception.status, 429)
+        self.assertIsNotNone(context.exception.headers.get('retry-after'))
```

Comment:
Would it be possible to store `retry-after` header value in the error object as well? That way it would be consistent with our other SDKs.